tests/model_inheritance/models.py

```diff
@@ -186,3 +188,24 @@ class Child(Parent):
 
 class GrandChild(Child):
     pass
+
+
+# Diamond inheritance
+class CommonAncestor(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
```suggestion
    pass
```
My advice/recommendation is to keep the tests & test fixtures as specific as it needs to be and no more, ie minimal without extraneous details.  This helps keep the testing focused & scientific. 😊

Reply:
Whoops the ticket reports that this is an issue with having a default, my recommendation still applies though as we just need to test with a default.  Having a default of 1 for all rows also proves that there's no duplicate rows.

This works and also shows the same exception reported (unique constraint failure on CommonAncestor.id):

```suggestion
    id = models.IntegerField(primary_key=True, default=1)
```